examples/layers/JSONLayers/Denomination.json

```diff
@@ -0,0 +1,27 @@
+{
+  "type": "color",
+  "url": "https://wxs.ign.fr/an7nvfzojv5wa96dsga5nk8w/geoportail/wmts",
+  "protocol": "wmts",
+  "id": "Denomination",
+  "title": "Geographical denomination in France",
+  "visible": false,
```

Comment:
apparently this doesn't work with the current code :-( Let's remove these `visible: false`. Actually, because they were all visible, I discovered a bug, so maybe it's actually better to have them all! (sorry to backtrack on that)

Reply:
From what I've seen, they are not visible, but the dat.gui panel is not showing the correct status. What is the bug you discovered ? It sometimes doesn't load correctly the tiles for my part, but that's not always the case so I don't really know.

Reply:
I pushed a fix for the GUITools in #598